awesome_collections.md

```diff
@@ -0,0 +1,13 @@
+# Awesome Hugging Face Collections
+
+<p align="center"><em>🎃 Note as part of [Hacktoberfest 2023](https://hacktoberfest.com/), we're looking for contributions to this page! If you have a collection you'd like to add, please submit a PR! See this issue for more context! #TODO add issue link 🎃</em></p>
+
+[Hugging Face Collections](https://huggingface.co/docs/hub/collections) provide a way of curating repositories from the Hub (Models, Datasets, Spaces and Papers) on a dedicated page. This page collects some examples of awesome collections! 
+
+## Awesome Hand Curated Hugging Face Collections
+- [Historic Multilingual Language Models](https://huggingface.co/collections/stefan-it/%F0%9F%93%9A-historic-multilingual-language-models-64f9c4f8383bbd73dddd2240) a Collection of Historic Multilingual Language Models. 
+- [Protein Design & Protein Structure Prediction](https://huggingface.co/collections/simonduerr/protein-design---protein-structure-prediction-64f9c6fda9295717466dbe8f) Interactive Demos that can be used for protein structure prediction using AlphaFold2 or RoseTTAfold2, prediction of small metal ions.
```

Comment:
```suggestion
- [Historic Multilingual Language Models](https://huggingface.co/collections/stefan-it/%F0%9F%93%9A-historic-multilingual-language-models-64f9c4f8383bbd73dddd2240) (by [stefan-it](https://huggingface.co/stefan-it)) a Collection of Historic Multilingual Language Models. 
- [Protein Design & Protein Structure Prediction](https://huggingface.co/collections/simonduerr/protein-design---protein-structure-prediction-64f9c6fda9295717466dbe8f) (by [simonduerr](https://huggingface.co/simonduerr)) Interactive Demos that can be used for protein structure prediction using AlphaFold2 or RoseTTAfold2, prediction of small metal ions.
```

I would add a `by stefan-it` by a url to user profile, just to add more recognization to the awesome contributions (and it's already the case in other sections)